themes/default/content/docs/intro/pulumi-service/projects-and-stacks.md

```diff
@@ -64,6 +64,147 @@ To view an organization's stacks:
 1. To view a stacks details select the name of the stack.
 1. To view a specific stack update, navigate to **Activity** and select it from the list.
 
+### Stack README
+
+To add a README to a stack:
+
+1. Set Stack Output named `readme` to the value of your templated Stack README file. In the example below, we've called the file `Pulumi.README.md`.
+2. Create a README template for the Stack.
+3. Run `pulumi up` on that Stack.
+4. Open the Pulumi Service UI, navigate to Projects and then the Stack you have updated. Once on the Stack page you will see the README tab with your README file.
+
+Examples for adding the Stack Output `readme` to a Pulumi program:
+
+{{< chooser language "typescript,python,go,csharp,java" / >}}
+
+{{% choosable language typescript %}}
```

Comment:
I'm not sure, I have only ever used choosable, what is the other approach?

Reply:
There's a new `<pulumi-chooser>` and `<pulumi-choosable>` element (component?) that I think 
@cnunciato added a couple weeks ago.

I don't know more than that we made sure that new docs for Java & YAML used those, unfortunately.

Reply:
@AaronFriel Actually what @meagancojocar has is right -- the `choosable` shortcode just wraps those custom elements (which aren't new; they've been around for a couple of years now), and we should use the shortcode expressions for them in any Markdown files we write. I'm guessing you're thinking of when I removed the shortcodes from our resource-docs _templates_ and replaced them with the raw elements -- I did that because those docs were so large that the perf cost of building them with shortcodes was too high. In general, unless you're working in a Hugo layout file (e.g., one with a `.html` extension), you shouldn't have to write any angle-bracketed choosers or choosables.